newproject.sh

```diff
@@ -169,14 +223,26 @@ echo "✅  Completed"
 echo "=> 🔎 Replacing package and package name within files..."
 PACKAGE_NAME_ESCAPED="${packagename//./\.}"
 OLD_PACKAGE_NAME_ESCAPED="${OLD_PACKAGE//./\.}"
-LC_ALL=C find $WORKING_DIR/$NAME_NO_SPACES -type f -exec sed -i "" "s/$OLD_PACKAGE_NAME_ESCAPED/$PACKAGE_NAME_ESCAPED/g" {} +
-LC_ALL=C find $WORKING_DIR/$NAME_NO_SPACES -type f -exec sed -i "" "s/$OLD_NAME/$NAME_NO_SPACES/g" {} +
+if [[ "$OSTYPE" == "darwin"* ]] #MacOS
```

Comment:
Just curious why you had to add this, what issue did you run in to? 🙏

Reply:
@Tuubz The original command was working well when I execute the script on local. Unfortunately, the machine on CI is running with Ubuntu instead, and the `find sed` command doesn't work, you can see the fail run [here](https://github.com/nimblehq/android-templates/runs/4542929431?check_suite_focus=true) so that's why I had to update to detect the OS to execute the command accordingly.